eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Serialisation/Tripping.hs

```diff
@@ -8,160 +8,155 @@
+--   (src -> Either target (BSL.ByteString, src)) ->
+--   src ->
+--   Property
+-- trippingF f x = runIdentity $ trippingM f' (===) x
+--   where
+--     f' a = pure (f a, toLazyByteString $ toCBOR a)
 
-trippingM ::
-  (Monad m, Show target) =>
-  (src -> m (Either target (BSL.ByteString, src), BSL.ByteString)) ->
-  (src -> src -> Property) ->
-  src ->
-  m Property
-trippingM f g x = do
-  res <- f x
-  pure $ case res of
-    (Right (remaining, y), _)
-      | BSL.null remaining ->
-          g x y
-    (Right (remaining, _), _) ->
-      counterexample
-        ("Unconsumed trailing bytes:\n" <> BSL.unpack remaining)
-        False
-    (Left stuff, bs) ->
-      counterexample
-        ( concat
-            [ "Failed to decode: ",
-              show stuff,
-              "\nbytes: ",
-              show (Base16.encode bs)
-            ]
-        )
-        False
+-- trippingM ::
+--   (Monad m, Show target) =>
+--   (src -> m (Either target (BSL.ByteString, src), BSL.ByteString)) ->
+--   (src -> src -> Property) ->
+--   src ->
+--   m Property
+-- trippingM f g x = do
+--   res <- f x
+--   pure $ case res of
+--     (Right (remaining, y), _)
+--       | BSL.null remaining ->
+--           g x y
+--     (Right (remaining, _), _) ->
+--       counterexample
+--         ("Unconsumed trailing bytes:\n" <> BSL.unpack remaining)
+--         False
+--     (Left stuff, bs) ->
+--       counterexample
+--         ( concat
+--             [ "Failed to decode: ",
+--               show stuff,
+--               "\nbytes: ",
+--               show (Base16.encode bs)
+--             ]
+--         )
+--         False
 
-trippingAnn ::
-  ( Eq t,
-    Show t,
-    ToCBOR t,
-    FromCBOR (Annotator t)
-  ) =>
-  t ->
-  Property
-trippingAnn = trippingF roundTripAnn
+-- trippingAnn ::
+--   ( Eq t,
+--     Show t,
+--     ToCBOR t,
+--     FromCBOR (Annotator t)
+--   ) =>
+--   t ->
+--   Property
+-- trippingAnn = trippingF roundTripAnn
 
-trippingAnnWithTwiddling :: (Twiddle t, FromCBOR (Annotator t)) => (t -> t -> Property) -> t -> Property
-trippingAnnWithTwiddling comp x = property $ trippingM roundTripAnnWithTwiddling comp x
+-- trippingAnnWithTwiddling :: (Twiddle t, FromCBOR (Annotator t)) => (t -> t -> Property) -> t -> Property
+-- trippingAnnWithTwiddling comp x = property $ trippingM roundTripAnnWithTwiddling comp x
 
-tripping :: (Eq src, Show src, ToCBOR src, FromCBOR src) => src -> Property
-tripping = trippingF roundTrip
+-- tripping :: (Eq src, Show src, ToCBOR src, FromCBOR src) => src -> Property
+-- tripping = trippingF roundTrip
 
-trippingWithTwiddling :: (Twiddle src, FromCBOR src) => (src -> src -> Property) -> src -> Property
-trippingWithTwiddling comp x = property $ trippingM roundTripWithTwiddling comp x
+-- trippingWithTwiddling :: (Twiddle src, FromCBOR src) => (src -> src -> Property) -> src -> Property
+-- trippingWithTwiddling comp x = property $ trippingM roundTripWithTwiddling comp x
```

Comment:
dead code, or  turned off temporarily?

Reply:
Yes, I intentionally commented it out. There will be a subsequent PR that fixes this part properly. I am partially done with its implementation too. These twiddler tests were skipped anyways, so I didn't see any harm in commenting them out in this PR.